spec/FilesController.spec.js

```diff
@@ -0,0 +1,27 @@
+var FilesController = require('../src/Controllers/FilesController').FilesController;
+var Config = require("../src/Config");
+
+// Small additional tests to improve overall coverage
+describe("FilesController",()=>{
+  
+  it("should properly expand objects", (done) => {
```

Comment:
This is already covered by the integration tests that are run through Parse.File.save